src/app/cluster/cluster-details/constraints/theme.scss

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@mixin theme-constraints-component($colors) {
+  .row-disabled {
+    &:hover {
+      background-color: transparent !important;

```

Comment:
Can we avoid using `!important` here? If that's too complex or there is `!important` somewhere above then ignore this comment.